package.json

```diff
@@ -106,7 +106,7 @@
     "typescript": "3.x",
     "url-loader": "^0.5.7",
     "webpack": "^5.24.2",
-    "webpack-cli": "^4.5.0",
+    "webpack-cli": "^4.7.2",
```

Comment:
What was this for?

Reply:
Tbh, not really sure. Probably must've happened when I pulled or installed some dependancies. I'll revert this for now!